src/swagger/auth.swagger.js

```diff
@@ -104,3 +104,26 @@
  *       '401':
  *         description: Incorrect credentials.
  */
+/**
+ * @swagger
+ * /api/auth/logout:
```

Comment:
I'm unable to test this using the swagger documentation. The authorization header field is disabled.

Reply:
Do try out, the field will become available.

Reply:
On my side it is working

Reply:
Got it to work now

Reply:
Add the protected route to the swagger definition so I can test the route also.

Reply:
That's good to hear.

Reply:
> Add the protected route to the swagger definition so I can test the route also.

Added it. you can test